R/human_infection.R

```diff
@@ -371,25 +381,24 @@ calculate_treated <- function(
 #' @description
 #' Schedule infections in humans after the incubation period
 #' @param events a list of all of the model events
+#' @param patent_infections bitset of patent-level infected humans (P.v only)
 #' @param clinical_infections bitset of clinically infected humans
 #' @param treated bitset of treated humans
 #' @param infections bitset of infected humans
 #' @param parameters model parameters
 #' @noRd
 schedule_infections <- function(
```

Comment:
We don't use any of these variables again after schedule_infections right? Let's avoid copying them loads of times.

Hopefully, your test will confirm that this works.